src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -166,6 +169,29 @@ private String getRandomVideo(ArrayList<String> playlistVideos) {
     int playlistSize = playlistVideos.size();
     int index = randomGenerator.nextInt(playlistSize);
     String videoId = playlistVideos.get(index);
+    
+    try {
+      getVideoInformation(videoId);
+    } catch (Exception e) {
+
+    }
+
     return videoId;
   }
+
+  private void getVideoInformation(String videoId) throws GeneralSecurityException, IOException, GoogleJsonResponseException {
```

Comment:
Either change this to getVideoInfo or change getPlaylistInfo to getPlaylistInformation, for consistency

Reply:
Actually, I think a better name for this method would be setVideoInfo/setVideoInformation

Reply:
You might consider making this two methods. One to retrieve the video information and one to send it to the datastore.